README.md

```diff
@@ -11,33 +11,30 @@ Our roadmap for the ReactJS Boilerplate is where you can learn about what featur
     - Optimisation of Minify code in production mode
 - Coding Standards and Guidelines (Statically analyzes our codes)
   - [ ] ESLint with our rules
-  - [ ] Prettier
-- Testing framework
-  - [x] Jest
-  - [ ] React Testing Library
-  - [ ] React Hooks Testing Library
-  - [ ] Storybook
-    - [x] base
-    - [ ] msw
-    - [ ] state management
-    - [ ] router
-    - [ ] css
-    - [ ] i18n
-  - [ ] E2E
-    - [x] Playwright
-    - [ ] Cypress
+  - [x] Prettier
 - Mock API server
   - [x] msw
 
+### Testing
+
+- Visual regression testing
+  - [x] Storybook
+- Unit testing
+  - [x] Jest
+  - [x] React Testing Library
+  - [ ] React Hooks Testing Library
```

Comment:
We can tick after merging the UT PR

Reply:
Now that the PR #206 has been merge, I think it can tick.
[React Hooks Testing Library](https://github.com/testing-library/react-hooks-testing-library) is not used so not ticked.

Reply:
You're right! We do not need to use [React Hooks Testing Library](https://github.com/testing-library/react-hooks-testing-library) anymore